boards/native/board_init.c

```diff
@@ -66,6 +66,10 @@ VFS_AUTO_MOUNT(fatfs, VFS_MTD(mtd0_dev), VFS_DEFAULT_NVM(0), 0);
 #elif defined(MODULE_LWEXT4)
 VFS_AUTO_MOUNT(lwext4, VFS_MTD(mtd0_dev), VFS_DEFAULT_NVM(0), 0);
 
+/* host fs pass-through */
+#elif defined(MODULE_FS_NATIVE)
+VFS_AUTO_MOUNT(native, { FS_NATIVE_DIR }, VFS_DEFAULT_NVM(0), 0);
```

Comment:
```suggestion
VFS_AUTO_MOUNT(native, { .hostpath: FS_NATIVE_DIR }, VFS_DEFAULT_NVM(0), 0);
```

would it make it easier for users to find where the hostpath is coming from